ui-react/src/components/topBar/ReportProblemModal.tsx

```diff
@@ -34,17 +37,27 @@ function ReportProblemModal({ show, onClose }: ReportProblemModalProps) {
   const [sourceVersion, setSourceVersion] = useState<string>();
 
   useEffect(async () => {
-    const lines = await getLogsFolder();
-    setLogsFolder(lines);
+    const logFolder = await getLogsFolder();
     const version = await sourceCommitShort();
+    const lines = await getLastLines(10);
+
+    setLogsFolder(logFolder);
     setSourceVersion(version);
+    // TODO: num of lines in the getLastLines not working so I slice the result for now
```

Comment:
What is not working? getLastLines doesn't limit the number according to the parameter you provide?
If that is the case, please either fix the root cause or remove the parameter that is not working

Reply:
I Removed the parameter that is not working since it does not seem to me as an urgent issue for now.

Reply:
I don't know if it is not urgent... What if the log is very large?

Reply:
As long as I don't know how to debug the server side I'm pretty stuck with this.